File: JavaScript/01.JavaScript-Basic-February-2021/04.conditionalStatements-Exercise/07.worldSwimmingRecord.js
function wordSwimmingPool(input) {
    let record = Number(input[0]);
    let distance = Number(input[1]);
    let time1m = Number(input[2]);

    let total = distance * time1m;
    let water = Math.floor(distance / 15) * 12.5;
    let final = total + water;

    if (record <= final) {
        console.log(`No, he failed! He was ${(final - record).toFixed(2)} seconds slower.`);
    } else {
        console.log(`Yes, he succeeded! The new world record is ${final.toFixed(2)} seconds.`);
    }
}

wordSwimmingPool(["10464", "1500", "20"]);
wordSwimmingPool(["55555.67", "3017", "5.03"]);

File: Python/01.Python-Basic-April-2019/05.conditional_statements_exercise/07.world_swimming_record.py
import math

record_seconds = float(input())
dist = float(input())
time_seconds = float(input())

ivan_time = dist * time_seconds
delay = math.floor((dist / 15)) * 12.5

total = ivan_time + delay
need = abs(record_seconds - total)

if total < record_seconds:
    print(f'Yes, he succeeded! The new world record is {total:.2f} seconds.')
else:
    print(f'No, he failed! He was {need:.2f} seconds slower.')
